examples/unicast/unicast_static.ini

```diff
@@ -0,0 +1,168 @@
+[General]
```

Comment:
The other example's ini-files are all named _omnetpp.ini_, could we rename this for consistency?